tests/unit/neptune/new/internal/backends/test_nql.py

```diff
@@ -54,10 +54,10 @@ def test_attributes(self):
                     name="sys/state",
                     type=NQLAttributeType.EXPERIMENT_STATE,
                     operator=NQLAttributeOperator.EQUALS,
-                    value="running",
+                    value="Active",
```

Comment:
This shouldn't be updated (calling an filtering API)

Reply:
Hm, I assumed that we adapt to some new API.